testing/btest/scripts/base/frameworks/storage/sqlite-cluster.zeek

```diff
@@ -0,0 +1,87 @@
+# @TEST-DOC: Tests SQLite storage in a cluster environment
+
+# @TEST-PORT: BROKER_PORT1
+# @TEST-PORT: BROKER_PORT2
+# @TEST-PORT: BROKER_PORT3
+
+# @TEST-EXEC: btest-bg-run manager-1 ZEEKPATH=$ZEEKPATH:.. CLUSTER_NODE=manager-1 zeek -b %INPUT
+# @TEST-EXEC: btest-bg-run worker-1  ZEEKPATH=$ZEEKPATH:.. CLUSTER_NODE=worker-1 zeek -b %INPUT
+# @TEST-EXEC: btest-bg-run worker-2  ZEEKPATH=$ZEEKPATH:.. CLUSTER_NODE=worker-2 zeek -b %INPUT
+# @TEST-EXEC: btest-bg-wait -k 5
+# @TEST-EXEC: btest-diff worker-1/.stdout
+# @TEST-EXEC: btest-diff worker-2/.stdout
+# @TEST-EXEC:
+
+@load base/frameworks/storage/sync
+@load base/frameworks/cluster
+@load policy/frameworks/storage/backend/sqlite
+@load policy/frameworks/cluster/experimental
+
+# @TEST-START-FILE cluster-layout.zeek
+redef Cluster::nodes = {
+	["manager-1"] = [$node_type=Cluster::MANAGER, $ip=127.0.0.1, $p=to_port(getenv("BROKER_PORT1"))],
```

Comment:
Mind making this manager rather than manager-1 to avoid churn if #4472  makes it?

Reply:
It's approved now it you want to merge it, and I'll adapt this test to that.